pkg/node/provisioners/drive_provisioner.go

```diff
@@ -227,14 +228,16 @@ func (d *DriveProvisioner) GetVolumePath(vol *api.Volume) (string, error) {
 	// For volumes that are in Published status, ignore partition re-read and
 	// get the partition number directly.
 	var partNum string
-	if vol.CSIStatus == apiV1.Published {
-		if partNum, err = d.partOps.GetPartitionNameByUUID(device, volumeUUID); err != nil {
-			return "", fmt.Errorf("get part name by uuid %s for device:%s : %w", volumeUUID, device, err)
-		}
-	} else {
-		if partNum, err = d.partOps.SearchPartName(device, volumeUUID); err != nil {
+	if partNum, err = d.partOps.SearchPartName(device, volumeUUID); err != nil {
+		// blockdev command run return exec.ExitError
+		if _, ok := err.(*exec.ExitError); ok && vol.CSIStatus == apiV1.Published {
```

Comment:
blockdev can return a few type of errors, we need to skip only disk is busy

Reply:
Hi  @yurkov-anton,  now I checked the exec_posix.go code,   we can use the exit status code to judge this instead of stderr output.   the exitError.ProcessState.ExitStatus will catch the exit status code. 
do you think it is ok ?

Reply:
As I know blockdev exit code doesn't say about type of error.
`err` contains error massage from blockdev. And we can check that `err` contains `Device or resource busy` part.

Reply:
Thank you @yurkov-anton for the reminder,  I found exitError contains Stderr, that's what we need here. updated code,  please help take a look again.